src/store/mutations/mutations-model.ts

```diff
@@ -1,5 +1,7 @@
-import { CertifyStatementIF, CertifyIF, IncorporationAddressIf, NameRequestIF,
-  BusinessContactIF, OrgPersonIF, ShareClassIF } from '@/interfaces'
+import {
+  CertifyStatementIF, CertifyIF, IncorporationAddressIf, NameRequestIF,
+  BusinessContactIF, OrgPersonIF, ShareClassIF, DateTimeIF
+} from '@/interfaces'
```

Comment:
If we are, in fact, importing all the IFs, can we use the syntax to import all?

Reply:
Yea, good idea!

Reply:
We could do this `import * as Interfaces from '@/interfaces' but would then have to prefix every interface we use as `Interfaces.SomeInterface`. 

Prefer longer import or a little more code?

Reply:
I prefer the longer import.

Reply:
cool, me as well :)